### PR TITLE
RDAPI-21101 Release change log updates

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Ticket: https://jira.axway.com/browse/RDAPI-21101

https://deploy-preview-1430--axway-open-docs.netlify.app/docs/apim_reference/allreleaseschangelog/

Updating links that were broken. Added link to Webliv homepage at end instead.

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_reference/allReleasesChangeLog.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ List of the major changes made in each release of API Gateway, API Manager, and
 |                                                                                                |Undesirable cipher suites disabled by default when using SSL/TLS. | | |
 |                                                                                                |Endpoint identification algorithms for LDAPS enabled by default. | | |
 |                                                                                                |Fields that contain confidential information are no longer returned in some API calls.| | |
-|[7.7 SP2](https://delivery.axway.int/GUI/file/GUI_file_readme.php?3i3aljnfkbsu3pn69e0s7ga307&fileId=74993&readmeId=38812&pdtId=394)|OpenSSL updated to 1.0.2t-fips to fix security vulnerabilities. | |21/12/2019 |
-|[7.7 SP1](https://delivery.axway.int/GUI/file/GUI_file_readme.php?3i3aljnfkbsu3pn69e0s7ga307&fileId=73780&readmeId=38290&pdtId=394)| | |29/08/2019 |
+|7.7 SP2*|OpenSSL updated to 1.0.2t-fips to fix security vulnerabilities. | |21/12/2019 |
```
Comment (andreamussap) [inline content/en/docs/apim_reference/allReleasesChangeLog.md:45]: @Bcolemanaxway why did you replace the links with * ?

Comment (andreamussap) [reply content/en/docs/apim_reference/allReleasesChangeLog.md:45]: Ok, I can see this at the bottom (For versions marked with * release notes can be found on [Webliv](https://delivery.axway.int)) but this is not working - https://delivery.axway.int

`This site can’t be reacheddelivery.axway.int’s server IP address could not be found.`

Comment (andreamussap) [reply content/en/docs/apim_reference/allReleasesChangeLog.md:45]: I've tested that you can find the service packs on the support site, so I'll update the link to there ok??

https://support.axway.com/en/search/index/dispatch/1?type=All&product=324&version=2921&q=service+pack+5

Comment (Bcolemanaxway) [reply content/en/docs/apim_reference/allReleasesChangeLog.md:45]: Yeah new link is good too @andreamussap

**Review by andreamussap — APPROVED** [review]

**Pull request merged**